### PR TITLE
Update .coveragerc

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -1,3 +1,7 @@
 [report]
-include =
```
Comment (tisto) [inline .coveragerc]: @hvelarde could you elaborate why this didn't work? An omit statement is harder to maintain if people add stuff. An include is a better solution in my option (only if it works of course).

Comment (hvelarde) [inline .coveragerc]: let me make a few tests; I think the right option is neither one, but [source](https://coverage.readthedocs.io/en/coverage-4.2/source.html).

**Comment by hvelarde** [comment]
@tisto I tested `source` but it did not work so `include` seems to be fine; may I ask why you are omitting `test` and `upgrades`?

**Comment by tisto** [comment]
@hvelarde the only thing this pull request now does is excluding the __init__.py on the first level of the package. Not sure why we should do that.

**Pull request closed without merging**